tests/monster_test.fbs

```diff
@@ -31,6 +31,11 @@ enum Race:byte {
   Elf,
 }
 
+enum LongEnum:ulong (bit_flags) {
+  LongOne = 1,
+  LongTwo = 2,
```

Comment:
can we set this value to be > 2^32 so we can see how it handles values that are larger than a short?